modules/chatops4s-examples/src/main/scala/chatops4s/examples/slack/Main.scala

```diff
@@ -0,0 +1,66 @@
+package chatops4s.examples.slack
+
+import cats.effect.{ExitCode, IO, IOApp}
+import cats.implicits.*
+import chatops4s.*
+import chatops4s.slack.*
+import chatops4s.slack.models.SlackConfig
+import org.typelevel.log4cats.Logger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+object Main extends IOApp {
+
+  given logger: Logger[IO] = Slf4jLogger.getLogger[IO]
+
+  override def run(args: List[String]): IO[ExitCode] = {
+    val config = SlackConfig(
+      botToken = sys.env.getOrElse("SLACK_BOT_TOKEN", "xoxb-your-bot-token-here"),
+      signingSecret = sys.env.getOrElse("SLACK_SIGNING_SECRET", "your-signing-secret-here"),
+      port = sys.env.get("PORT").flatMap(_.toIntOption).getOrElse(3000)
+    )
```

Comment:
why do you have those here and also something in `.conf` file? It hink one of them is redundant